# ARG007: Specified property type is not supported

## Summary
A property annotated with an argument attribute has a type that is not supported by the ArgumentParser.

## Cause
This error occurs when a property with an `Option`, `Flag`, or `Positional` attribute has a type that the parser cannot automatically convert from string arguments.

## How to fix
Use one of the supported property types for your command-line arguments:

### Supported types:
- `string`
- `int`/`System.Int32`
- `bool`/`System.Boolean`
- `double`/`System.Double`
- `float`/`System.Single`
- `long`/`System.Int64`
- `short`/`System.Int16`
- `decimal`/`System.Decimal`
- `byte`/`System.Byte`
- `sbyte`/`System.SByte`
- `char`/`System.Char`
- `uint`/`System.UInt32`
- `ulong`/`System.UInt64`
- `ushort`/`System.UInt16`
- `Guid`/`System.Guid`
- `Uri`/`System.Uri`
- `TimeSpan`/`System.TimeSpan`
- `DateTime`/`System.DateTime`

```csharp
// Valid property types:
[Option("o", "Output")]
public string OutputPath { get; set; }

[Flag("v", "Verbose")]
public bool IsVerbose { get; set; }

[Positional(0, "Count of items")]
public int Count { get; set; }

// Invalid property types would be custom classes or unsupported types:
[Option("c", "Config")]
public MyConfiguration Config { get; set; } // ERROR: Custom class not supported
```