# ARG005: Invalid positional argument position

## Summary
A positional argument has an invalid position value.

## Cause
This error occurs when a `Positional` attribute specifies a position value that is negative or out of range.

Positional arguments must have positions that start at 0 and form a continuous sequence (0, 1, 2, ...).

## How to fix
Ensure that positional arguments:
1. Have non-negative position values (0 or greater)
2. Form a continuous sequence starting from 0

```csharp
// Before - Error:
[Positional(-1, "Invalid position")] // Negative position
public string InvalidArg { get; set; }

[Positional(2, "Skipped position")] // Skips position 1
public string SkippedArg { get; set; }

// After - Fixed:
[Positional(0, "First argument")]
public string FirstArg { get; set; }

[Positional(1, "Second argument")]
public string SecondArg { get; set; }

[Positional(2, "Third argument")]
public string ThirdArg { get; set; }
```